project.clj

```diff
@@ -16,6 +16,8 @@
                  [io.netty/netty-transport ~netty-version]
                  [io.netty/netty-transport-native-epoll ~netty-version :classifier "linux-x86_64"]
                  [io.netty/netty-transport-native-epoll ~netty-version :classifier "linux-aarch_64"]
+                 [io.netty/netty-transport-native-kqueue ~netty-version :classifier "osx-x86_64"]
+                 [io.netty.incubator/netty-incubator-transport-native-io_uring "0.0.12.Final" :classifier "linux-x86_64"]
```

Comment:
Is the classifier required? Or conversely, are there any ARM versions of io_uring we need to add?

Reply:
Good question...

According to the documentation on GitHub, it was the _recommendation_ [1].
But, if you look at Maven registry, we are also behind in terms of dependency [2] and the classifier is not mentioned.

I will try to remove it and bump to version "0.0.16.Final".

[1] : https://github.com/netty/netty-incubator-transport-io_uring
[2] : https://mvnrepository.com/artifact/io.netty.incubator/netty-incubator-transport-native-io_uring/0.0.16.Final

Reply:
But yes, there is an ARM version

```
x86-64 / aarch64 processor
```

Reply:
Yes, the classifier is mandatory, otherwise you end up with a `jar` containing only some metadata. [1]
I will add both classifier.

[1] : https://repo1.maven.org/maven2/io/netty/incubator/netty-incubator-transport-native-io_uring/0.0.16.Final/